static/js/components/GroupPageManageUserButtons.jsx

```diff
@@ -63,49 +65,60 @@ const ManageUserButtons = ({ group, loadedId, user, isAdmin }) => {
     }
   };
 
+  const handleDelete = () => {
+    dispatch(
+      groupsActions.deleteGroupUser({
+        userID: user.id,
+        group_id: group.id,
+      })
+    );
+    history.goBack();
```

Comment:
Why is the back navigation necessary here?

Reply:
@stefanv Because once you are out of the group, the group page does not exist for you anymore :)

Reply:
Maybe it should be some kind of if statement?

Reply:
I wonder if we shouldn't splash a pop up warning saying do you really want to leave etc? This is a pretty big change and we don't want people to click this button without understanding the implication.

Reply:
@profjsb @stefanv Is there an example of this in the app already?

Reply:
@profjsb @stefanv I just didn't know what to grep for. Indeed that confirmation exists!